packages/esm-providers-app/src/workspace/provider-form.workspace.tsx

```diff
@@ -421,21 +439,23 @@ const ProviderForm: React.FC<ProvideModalProps> = ({ closeWorkspace, provider, u
             )}
           />
         </Column>
-        <Controller
-          name="nationalid"
-          control={control}
-          render={({ field }) => (
-            <TextInput
-              {...field}
-              id="form__national__id"
-              placeholder={t('nationalIdPlaceholder', 'Enter National ID')}
-              labelText={t('nationalID', 'National ID*')}
-              disabled
-              invalid={!!errors.nationalid}
-              invalidText={errors.nationalid?.message}
-            />
-          )}
-        />
+        {provider && (
+          <Controller
+            name="nationalid"
+            control={control}
+            render={({ field }) => (
+              <TextInput
+                {...field}
+                id="form__national__id"
```

Comment:
```suggestion
                id="form__national__id"
```
We should be following standard convention here e.g `formNationalId`